admin_manual/installation/source_installation.rst

```diff
@@ -359,13 +359,38 @@ SELinux-enabled distributions such as Fedora and CentOS.
 php.ini Configuration Notes
 ---------------------------
 
-.. IMPORTANT:: 
-   From ownCloud 9.1.2, ensure that you have `session.auto_start
-   = 0 <php_session_autostart` in your configuration. If not, you may have
-   issues logging in to ownCloud via the WebUI.
+From ownCloud 9.1.2, several core PHP settings have to be configured correctly,
+otherwise ownCloud may not work properly. These are all listed here.
 
-Keep in mind that changes to ``php.ini`` may have to be configured in more than one 
-ini file. This can be the case, for example, for the ``date.timezone`` setting.
+session.auto_start && enable_post_data_reading
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Ensure that `session.auto_start`_ and `enable_post_data_reading`_ are set to
```

Comment:
That's strange. There's a previous ticket which indicated that both needed to be set to 0 or `false`.